pkg/cmd/config/config.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/profile"
+
+	"github.com/redhat-developer/app-services-cli/internal/config"
+	"github.com/redhat-developer/app-services-cli/pkg/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/logging"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+	"github.com/spf13/cobra"
+)
+
+type Options struct {
+	IO        *iostreams.IOStreams
+	Config    config.IConfig
+	Logger    func() (logging.Logger, error)
+	localizer localize.Localizer
+}
+
+func NewConfigCommand(f *factory.Factory) *cobra.Command {
+	opts := &Options{
+		IO:        f.IOStreams,
+		Config:    f.Config,
+		Logger:    f.Logger,
+		localizer: f.Localizer,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "config",
+		Short:   opts.localizer.MustLocalize("config.cmd.shortDescription"),
+		Long:    opts.localizer.MustLocalize("config.cmd.longDescription"),
+		Example: opts.localizer.MustLocalize("config.cmd.example"),
+	}
+
+	devPreview := &cobra.Command{
+		Use:       "devPreview",
```

Comment:
```suggestion
		Use:       "dev-preview",
```

Id prefer to use kebab-case for commands.

Reply:
WOW. Totally forgot! 
Actually our ruleset should catch it once we add regexp rule. CC @ankithans